### ISSUE TITLE
Accessibility finding: Accredited Representatives page missing unique page title

### ISSUE THREAD

**Issue by emisparkles** [department-of-veterans-affairs/va.gov-team#133927]
### Checklist item

Automated-001	Axe DevTools has been executed for every page

### Severity level

3, Medium. Should be fixed in 1-3 sprints post-launch.

### Page(s)/url(s)

Accredited representative or VSO
[staging.va.gov/profile/accredited-representative](https://staging.va.gov/profile/accredited-representative)

### Details

The Accredited Representative or VSO page in the VA.gov Profile does not have a unique page title. Instead, the page title is carried over from the previous page the user visited.

<img width="909" height="702" alt="Image" src="https://github.com/user-attachments/assets/ac17bb15-a5bd-4175-9c0b-58b076829941" />

### Reproduction steps

1. Log into staging.va.gov
2. Navigate to any profile page (not Accredited rep page) & note the page title
3. Using the side nav or URL bar, navigate to the Accredited representative page
4. Page title will be the same as previous page

### Proposed solution or next steps

Page title should be updated to "Accredited Representative or VSO | Veterans Affairs".

Add something like the following to [AccreditedRepresentative.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/personalization/profile/components/accredited-representative/AccreditedRepresentative.jsx):

```javascript
document.title = `Accredited Representative or VSO | Veterans Affairs`;
```

**Comment by gileswells** [comment]
> Noting that this is an issue for all page titles in the `Edit contact preferences` flow as well.

@erinrwhite all of the title tags and h1 tags across all of Profile had their capitalization specifically audited in the above PR that @emisparkles mentioned.

**Comment by jasonday** [comment]
> > Noting that this is an issue for all page titles in the `Edit contact preferences` flow as well.
> 
> [@erinrwhite](https://github.com/erinrwhite) all of the title tags and h1 tags across all of Profile had their capitalization specifically audited in the above PR that [@emisparkles](https://github.com/emisparkles) mentioned.

Tagging @KKitagawa-Bosch who is now handling IA

**Comment by PatSiebenlist** [comment]
Need to validate this was resolved after merge
